src/starbase/config.ts

```diff
@@ -20,6 +20,24 @@ const integrationSchema: Schema = {
   additionalProperties: true,
 };
 
+
+const storageSchema: Schema = {
+  type: 'object',
+  properties: {
+    engine: { type: 'string' },
+    config: { type: 'object' },
+  },
+  required: ['engine']
+}
+
+const overallSchema: Schema = {
```

Comment:
More opinionated but not a big fan of `overallSchema` naming. how about something more descriptive like `configSchema`?

Reply:
Updated to configSchema.